### DISCUSSION TITLE
Labeled_data_release_v0.0.1

### DISCUSSION THREAD

**Discussion by Zzzzzhuzhiwei** [X-lab2017/open-digger#1330] [General]
The reason we separate the label data release is that many developers who are not familiar with OpenDigger's development may not be aware of the existence of such labeled data. By releasing it separately, we can help developers to better conduct data analysis in the open-source domain. The label data we are initially releasing is the company's labeled data, and we are assigning it the version number 0.0.1.

<hr /><em>This discussion was created from the release <a href='https://github.com/X-lab2017/open-digger/releases/tag/'>Labeled_data_release_v0.0.1</a>.</em>

**Comment by stevending1st** [comment]
Cooooooool, the url is https://github.com/X-lab2017/open-digger/releases/tag/v0.0.1
